examples/computer_vision/efficientdet_pytorch/adaptive.yaml

```diff
@@ -0,0 +1,102 @@
+description: efficientdet_adaptive
+hyperparameters:
+  global_batch_size: 44
+  min_loss_scale:
+    type: double
+    minval: 8
+    maxval: 64
+  sync_bn: 
+    type: categorical
+    vals: [True, False]
+  model_ema: True
+  amp: True
+  root: '/mnt/data'
```

Comment:
I don't see this being used in any non-configuration files. Ditto in const.yaml and distributed.yaml.